packages/sil-governance/src/components/Header/index.tsx

```diff
@@ -1,19 +1,48 @@
 import { Address } from "@iov/bcp";
 import { Electorate } from "@iov/bns";
 import { Block, CircleImage, Image, Typography } from "medulas-react-components";
-import React from "react";
+import React, { useMemo } from "react";
 
 import iovLogo from "../../assets/iov-logo-title.svg";
 import userIcon from "../../assets/user.svg";
+import { history } from "../../routes";
+import { SHOW_ELECTORATE_ROUTE } from "../../routes/paths";
 
 export const HEADER_HTML_ID = "header";
 
+const navigateToElectorate = (id: number): void => {
+  const target = `${SHOW_ELECTORATE_ROUTE}/${id}`;
+  if (history.location.pathname !== target) {
+    history.push(target);
+  }
+};
+
 interface Props {
   readonly address: Address;
   readonly electorates: readonly Electorate[];
 }
 
 const Header = ({ address, electorates }: Props): JSX.Element => {
+  const ElectorateLinks = useMemo(
+    () => (): JSX.Element => (
+      <React.Fragment>
+        {electorates.map((e, index) => (
+          <Block display="inline" key={e.id}>
+            <Typography inline link variant="body2" onClick={() => navigateToElectorate(e.id)}>
+              {e.title}
+            </Typography>
+            {index !== 1 && (
+              <Typography inline variant="body2">
+                ,{" "}
+              </Typography>
+            )}
```

Comment:
This is placed behind link 0, 2, 3, 4, 5, and so on. It only leads to the correct result for 2 links. What you want is to prepand this before every link other than 0.